Framework/Beamline/inc/MantidBeamline/ComponentInfo.h

```diff
@@ -66,8 +66,7 @@ class MANTID_BEAMLINE_DLL ComponentInfo {
   const int64_t m_sampleIndex = -1;
   DetectorInfo *m_detectorInfo; // Geometry::DetectorInfo is the owner.
   size_t m_scanCounts = 1;
-  Kernel::cow_ptr<std::vector<std::pair<int64_t, int64_t>>> m_scanIntervals{
-      nullptr};
+  std::vector<std::pair<int64_t, int64_t>> m_scanIntervals{{0, 1}};
```

Comment:
Maybe put a comment (doxygen `///`) here, explaining that `{0, 1}` is the default, i.e., 1 interval indicating "no scan".